.pre-commit-config.yaml

```diff
@@ -690,80 +561,81 @@ repos:
         pass_filenames: true
         exclude: >
           (?x)
-          ^airflow-core/src/airflow/ui/src/i18n/config\.ts$|
-          ^airflow-core/src/airflow/ui/openapi-gen/|
-          ^airflow-core/src/airflow/ui/public/i18n/locales/de/README\.md$|
+          ^airflow-core/docs/.*commits\.rst$|
```

Comment:
Reviewer Note: This is a re-sort of the grown list, only change is that I needed to add the new providers/.pre-commit-config.yaml as the term "master" was in.

Reply:
Cool